packages/rspack/src/lib/BannerPlugin.js

```diff
@@ -124,4 +124,4 @@ class BannerPlugin {
 	}
 }
 
-module.exports = BannerPlugin;
+export { BannerPlugin };
```

Comment:
In case of users who directly consumes `"webpack/lib/EnvironmentPlugin"`, should we support this?  What do you think? cc@hardfist

Reply:
can this plugin accessed by webpack.EnvironmentPlugin

Reply:
https://github.com/web-infra-dev/rspack/pull/3596/files#diff-0f776ba59eeb95c015bb94540359c4a7131f29fa6001ab35f864e64f2a7dc065R3, as you can see, yes.

Reply:
Do you mean webpack instance?

Reply:
yes，that is enough now